HPC/README.md

```diff
@@ -0,0 +1,114 @@
+# Getting started with HPC workloads on Graviton instances
+## Contents
```

Comment:
how about spack.io? are there any readily available binaries that can enable quick evaluation??

Reply:
We already have the binaries. I will generate PR for spack.  Once merged, we can post the links in this document.